modules/tinymce/src/themes/silver/test/ts/phantom/toolbar/ToolbarButtonsTest.ts

```diff
@@ -334,8 +337,9 @@ describe('phantom.tinymce.themes.silver.toolbar.ToolbarButtonsTest', () => {
     const body = hook.body();
     const component = hook.component();
     const store = hook.store();
-    const button4 = getButton('.button4-container .tox-mbtn');
+    hook.store().clear();
```

Comment:
```suggestion
    store.clear();
```

Reply:
:facepalm: I meant to do that before logging the PR. Thanks for the catch!